applications/nvidia_nim/nvidia_nim_imaging/README.md

```diff
@@ -0,0 +1,65 @@
+# NVIDIA NIM Imaging with Vista-3D
+
+This is a sample application demonstrates the use of Vista-3D NVIDIA Inference Microservice (NIM).
+
+The application instructs the Vista-3D NIM API to process the given dataset and then downloads and extracts the results of a segmentation NRRD file onto a local directory.
+
+### Quick Start
+
+1. Add API key in `nvidia_nim.yaml`
+2. `./dev_container build_and_run nvidia_nim_chat`
+
+## Configuring the sample application
+
+Use the `nvidia_nim.yaml` configuration file to configure the sample application:
+
+### Connection Information
+
+```
+nim:
+  base_url: https://integrate.api.nvidia.com/v1
+  api_key:
+
+```
+
+`base_url`: The URL of your NIM instance. Defaults to NVIDIA hosted NIMs.
+`api_key`: Your API key to access NVIDIA hosted NIMs.
+
+
+## Build and Run the sample application
+
+```
+# Build the Docker images from the root directory of Holohub
+./dev_container build_and_run nvidia_nim_imaging
+```
+
+## Display the Results
+
+In this section we will show how to view the sample data and segmentation results returned from Vista-3D.
+
+1. Decompress the sample data volume:
+   ```
+   gzip -d build/nvidia_nim_imaging/applications/nvidia_nim/nvidia_nim_imaging/sample.nii.gz
+   ```
```

Comment:
Can you confirm whether this step is necessary? `.nii.gz` is a standard medical image format, I would expect you can load `sample.nii.gz` into 3D Slicer and then 3D Slicer will handle the decompression.

Reply:
You were right, this step is not required.